nilearn/connectome/connectivity_matrices.py

```diff
@@ -193,7 +193,7 @@ def _geometric_mean(matrices, init=None, max_iter=10, tol=1e-7):
     return gmean
 
 
-def sym_to_vec(symmetric, discard_diagonal=False):
```

Comment:
I'm wondering if it's okay to remove  `sym_to_vec` completely. Could we put  a deprecation instead ?